src/apps/settings/src/app/views/Fonts/Browse.js

```diff
@@ -352,11 +355,27 @@ export default connect((state) => {
   return (
     <div className={styles.PageContainer}>
       <header className={styles.SearchContainer}>
-        <Search
-          className={styles.search}
+        <TextField
+          id="filled-search"
           placeholder="Search font"
-          // onSubmit={onSearch}
-          onKeyUp={(e) => onSearch(e.target.value)}
+          type="search"
+          variant="outlined"
+          fullWidth
+          size="small"
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <SearchIcon fontSize="small" />
+              </InputAdornment>
+            ),
+          }}
+          onChange={(evt) => {
+            let term = evt.target.value;
+
+            if (term === null) return;
+            onSearch(term);
+          }}
+          sx={{ width: "initial" }}
```

Comment:
Confused about this sx. Should the fullWidth not be passed as a prop in this case?